app/src/main/java/org/mozilla/fenix/share/ShareFragment.kt

```diff
@@ -71,7 +68,7 @@ class ShareFragment : AppCompatDialogFragment() {
         shareToAccountDevicesView =
             ShareToAccountDevicesView(view.devicesShareLayout, shareInteractor)
 
-        if (args.url != null && args.tabs == null) {
+        if (args.translucent) {
```

Comment:
Since you introduced a single concept `shareData`, couldn't this check just be `shareData.size == 1` instead of the caller needing to tell us that there is only one item?

Reply:
I wasn't sure if it was always the case that 1 item == translucent, which is why I used the explicit flag. I can take out the flag and just check the list size.

Reply:
According to the comments there, that's what I understand from it. If you chose to keep the flag, let's update those comments and use another name instead of `translucent`. I was confused what it was referring to.

Reply:
https://github.com/mozilla-mobile/fenix/issues/2754#issuecomment-532317276 indicates that the translucent scrim is for sharing from the browser fragment. Possible names: `overlay`, `fromBrowser`, `translucentScrim`?

Reply:
How about `showBrowser` or `showShareTabs` (depending on which way you want to flip the if-else)? Describing the action seems clearer to me than the thing it does internal, but that's just my opinion. 🤷‍♂

Reply:
Ah, I see now that the comment is in-accurate and based off an older implementation. So it confused the feature with the old impl? Sorry for making you do the comment clean up since it isn't related to your review.